hr_contract_update_overtime/README.rst

```diff
@@ -38,6 +38,9 @@ You can Update Overtime in different ways:
 -  Header button on Contract History form view
 -  Header button on Contract form view
```

Comment:
- Header button on Contract form view is visible if the user is responsible HR for the employee